src/modules/CRUD/components/Details/PropertyItem/Type/ArrayType/ArrayType.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getObjectOrderedValue, isTableObject } from '../../../../../services/utils';
+
+import Type from '..';
+import NoValue from '../../NoValue';
+
+const ArrayType = props => {
+  const {
+    display_value: displayValue,
+    schema,
+    schema: {
+      items: {
+        properties,
+        type,
+      },
+    },
+    t,
+    ui_schema: uiSchema,
+    ui_schema: {
+      items: {
+        'ui:order': UIOrder,
+      } = {},
+    } = {},
+  } = props;
```

Comment:
Why not destructuring directly in function argument?
*(I also suggest some inlining to help identifying structure)*

```jsx
const ArrayType = ({
  display_value: displayValue,
  schema,
  schema: { items: { properties, type } },
  t,
  ui_schema: uiSchema,
  ui_schema: { items: { 'ui:order': UIOrder } = {} } = {},
}) => {
  // …
```

or eventually:


```jsx
const ArrayType = ({
  display_value: displayValue,
  ui_schema: uiSchema,
  schema,
  t,
}) => {
  const { items: { properties, type } } = schema;
  const { items: { 'ui:order': UIOrder } = {} } = uiSchema;

  // …
```

This would be, I think, much more readable.